src/sparse_blas/backends/mkl_common/mkl_handles.hpp

```diff
@@ -0,0 +1,76 @@
+/***************************************************************************
+*  Copyright (C) Codeplay Software Limited
+*  Licensed under the Apache License, Version 2.0 (the "License");
+*  you may not use this file except in compliance with the License.
+*  You may obtain a copy of the License at
+*
+*      http://www.apache.org/licenses/LICENSE-2.0
+*
+*  For your convenience, a copy of the License has been included in this
+*  repository.
+*
+*  Unless required by applicable law or agreed to in writing, software
+*  distributed under the License is distributed on an "AS IS" BASIS,
+*  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+*  See the License for the specific language governing permissions and
+*  limitations under the License.
+*
+**************************************************************************/
+
+#ifndef _ONEMKL_SRC_SPARSE_BLAS_BACKENDS_MKL_COMMON_MKL_HANDLES_HPP_
+#define _ONEMKL_SRC_SPARSE_BLAS_BACKENDS_MKL_COMMON_MKL_HANDLES_HPP_
+
+// MKLCPU and MKLGPU backends include
+// This include defines its own oneapi::mkl::sparse namespace with some of the types that are used here: matrix_handle_t, index_base, transpose, uplo, diag.
+#include <oneapi/mkl/spblas.hpp>
+
+#include "sparse_blas/generic_container.hpp"
+
+namespace oneapi::mkl::sparse {
+
+// Complete the definition of incomplete types dense_vector_handle and dense_matrix_handle as there is no conflict with the oneMKL backends.
```

Comment:
"no conflict with oneMKL backends", or because "they don't exist in oneMKL backends yet"?

Reply:
I used your second suggestion and split the comments on multiple lines in https://github.com/oneapi-src/oneMKL/pull/500/commits/5c37ee30a4975789bbe8ab7f35632d7e2b205292
Not sure why clang-format does not split them.